### ISSUE TITLE
get_js_session raises an error if there is no running session

### ISSUE THREAD

**Issue by 0xIslamTaha** [threefoldtech/js-ng#108]
this [line](https://github.com/js-next/js-ng/blob/development/jumpscale/core/executors/tmux.py#L11) raises an error if there is no running session
```bash
JS-NG> j.core.executors.tmux.get_js_session()                                                                                                                                                                        
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/js-ng/jumpscale/core/executors/tmux.py", line 11, in get_js_session
    s = server.find_where({'session_name': JS_SESSION_NAME})
        │                                  └ 'js-ng'
        └ <libtmux.server.Server object at 0x7f29dcec6cc0>
  File "/root/.cache/pypoetry/virtualenvs/js-ng-py3.7/lib/python3.7/site-packages/libtmux/common.py", line 320, in find_where
    return self.where(attrs)[0]
           │          └ {'session_name': 'js-ng'}
           └ <libtmux.server.Server object at 0x7f29dcec6cc0>
  File "/root/.cache/pypoetry/virtualenvs/js-ng-py3.7/lib/python3.7/site-packages/libtmux/common.py", line 351, in where
    return list(filter(by, self.children))
                       │   └ <libtmux.server.Server object at 0x7f29dcec6cc0>
                       └ <function TmuxRelationalObject.where.<locals>.by at 0x7f29dc70eea0>
  File "/root/.cache/pypoetry/virtualenvs/js-ng-py3.7/lib/python3.7/site-packages/libtmux/server.py", line 188, in sessions
    return self.list_sessions()
           └ <libtmux.server.Server object at 0x7f29dcec6cc0>
  File "/root/.cache/pypoetry/virtualenvs/js-ng-py3.7/lib/python3.7/site-packages/libtmux/server.py", line 183, in list_sessions
    return [Session(server=self, **s) for s in self._sessions]
            │              │                   └ <libtmux.server.Server object at 0x7f29dcec6cc0>
            │              └ <libtmux.server.Server object at 0x7f29dcec6cc0>
            └ <class 'libtmux.session.Session'>
  File "/root/.cache/pypoetry/virtualenvs/js-ng-py3.7/lib/python3.7/site-packages/libtmux/server.py", line 173, in _sessions
    return self._list_sessions()
           └ <libtmux.server.Server object at 0x7f29dcec6cc0>
  File "/root/.cache/pypoetry/virtualenvs/js-ng-py3.7/lib/python3.7/site-packages/libtmux/server.py", line 153, in _list_sessions
    raise exc.LibTmuxException(proc.stderr)
          │                    └ <libtmux.common.tmux_cmd object at 0x7f29b30c3c18>
          └ <module 'libtmux.exc' from '/root/.cache/pypoetry/virtualenvs/js-ng-py3.7/lib/python3.7/site-packages/libtmux/exc.py'>
libtmux.exc.LibTmuxException: ['no server running on /tmp/tmux-0/default']

['no server running on /tmp/tmux-0/default']
JS-NG>  

```

**Comment by AhmedHanafy725** [comment]
This should be solved in #420

**Comment by siddiquagig** [comment]
Verified

![image](https://user-images.githubusercontent.com/25789724/89193350-e7ec2b00-d5b6-11ea-8e2b-05a3d45b85c1.png)
